### PR TITLE
clean up releases, move up docs, add Template Site

### PR DESCRIPTION
**Pull request by cvogt**
review by @d6y

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -70,10 +53,27 @@ title: Documentation
...
+* [Slick 3](third-party-slick-3)
+* [Slick 2](third-party-slick-2)
+* [Slick 1](third-party-slick-1)
+
+We cannot guarantee for the quality of the above, or that they represent our view of things.
+Please add more entries using a github pull request, if they can teach people something about Slick.
+
+## Books
+
+<a href="http://underscore.io/books/essential-slick/"><img src="{{ site.url }}/resources/images/essential-slick-cover.png" alt="Essential Slick book cover" title="Essential Slick book cover" height="100" /></a>
```
Comment (cvogt) [inline docs/index.md]: 100 might be too small. github rendering service seems down, so I can't check

Comment (d6y) [inline docs/index.md]: 100 looks fine to me.

File: docs/index.md [hunk 2]
```diff
@@ -1,38 +1,21 @@
+
```
Comment (d6y) [inline docs/index.md]: This additional blank line is causing a problem for Jekyll: the .html file isn't generated. If I remove the blank line, I can generate the site.

BTW, I'm running the site like this:

```
rm Gemfile
docker run --rm --label=jekyll --volume=$(pwd):/srv/jekyll -it -p 4000:4000 jekyll/jekyll:pages jekyll s --watch --base
url '' --force_polling
```

File: docs/third-party-slick-3.md [hunk 3]
```diff
@@ -33,6 +33,8 @@ and on [Stackoverflow](http://stackoverflow.com/questions/tagged/slick).
 
 ## Example projects
 
+- [Template Site](https://github.com/tetrao-eu/template-site) Skeleton of a web application based on Scala, SBT, Play Framework, Play2-auth, PostgreSQL, Slick, Slick-pg and Foundation [https://template-site.tetrao.eu/]
```
Comment (d6y) [inline docs/third-party-slick-3.md]: The link at the end of the line is rendering as...

selection_001

**Comment by cvogt** [comment]
@d6y fixed the issues

**Pull request merged**